xrpl4j-client/src/main/java/org/xrpl/xrpl4j/client/XrplClient.java

```diff
@@ -493,6 +497,20 @@ public AccountInfoResult accountInfo(AccountInfoRequestParams params) throws Jso
     return jsonRpcClient.send(request, AccountInfoResult.class);
   }
 
+  /**
+   * Return AccountNftsResult for an {@link Address}.
+   *
+   * @param account to get the NFTs for.
+   *
+   * @return {@link AccountNftsResult} containing list of accounts for an address.
+   * @throws JsonRpcClientErrorException If {@code jsonRpcClient} throws an error.
+   */
+  public AccountNftsResult accountNfts(Address account) throws JsonRpcClientErrorException {
+    return this.accountNfts(
+      AccountNftsRequestParams.builder().account(account).build()
+    );
+  }
+
   /**
    * Get the {@link AccountNftsResult} for the account specified in {@code params} by making an account_channels
```

Comment:
Is `account_channels` a typo?

```suggestion
   * Get the {@link AccountNftsResult} for the account specified in {@code params} by making an account_nfts
```

Reply:
It is